samples/analyze-streaming-automl-classification.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the `License`);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an `AS IS` BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(
+  path = 'YOUR_LOCAL_FILE',
+  projectId = 'YOUR_GCP_PROJECT',
+  modelId = 'YOUR_AUTOML_MODELID'
+) {
+  // [START video_streaming_automl_classification_beta]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const path = 'Local file to analyze, e.g. ./my-file.mp4';
+  // const modelId = 'autoMl model'
+  // const projectId = 'Your GCP Project'
+
+  const {
+    StreamingVideoIntelligenceServiceClient,
+  } = require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+
+  // Instantiates a client
+  const client = new StreamingVideoIntelligenceServiceClient();
+
+  // Streaming configuration
+  const modelPath = `projects/${projectId}/locations/us-central1/models/${modelId}`;
+  const configRequest = {
+    videoConfig: {
+      feature: 'STREAMING_AUTOML_CLASSIFICATION',
+      automlClassificationConfig: {
+        modelName: modelPath,
+      },
+    },
+  };
+  const readStream = fs.createReadStream(path, {
```

Comment:
We need a comment about video format for the input file here. And link to the help page if people get stuck.